python_modules/libraries/dagster-dbt/dagster_dbt_tests/conftest.py

```diff
@@ -8,7 +8,7 @@
 # ======= CONFIG ========
 DBT_EXECUTABLE = "dbt"
 TEST_PROJECT_DIR = file_relative_path(__file__, "dagster_dbt_test_project")
-DBT_CONFIG_DIR = os.path.join(TEST_PROJECT_DIR, "dbt_config")
```

Comment:
`profiles.yml` should live in the project directory anyways: https://docs.getdbt.com/docs/core/connection-profiles#advanced-customizing-a-profile-directory.

After this stack lands, we should clean up our test suite to follow dbt best practices, since some of this configuration is from dbt<1.0 days.